### DISCUSSION TITLE
模版更新后如何快速应用到我的任务

### DISCUSSION THREAD

**Discussion by NOwin111** [qd-today/templates#350] [Q&A] [answered]
因为使用了PT通用模板，所以一个模版应用的任务比较多，更新模版后如何方便的更新到对应的我的任务？

**Comment by a76yyyy** [comment, marked as answer]
现在暂时只支持替换法, 比较麻烦, 容易出错, 之后会考虑更方便的更新方式:

1. 公共模板点击订阅, 进入 `模板编辑` 页面, 不要点击 `保存`, 直接关闭页面
2. 在框架首页找到要更新的模板, 点击 `编辑`
3. 进入 `模板编辑` 页面后, 会提示 `您有一个正在编辑的 HAR 文件`, 选择 `打开`
4. 检查模板内容是否正确, 点击 `保存` 即可更新旧模板

**Reply by NOwin111** [reply]
好的，已经按照上面方法更新了。模板希望可以有一个版本号，用来判断模板的更新情况

**Reply by a76yyyy** [reply]
有时间我思考一下新的更新机制吧
